### PR TITLE
Few missing tests for bidiag

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

**Review by andreasnoack — APPROVED** [review]

File: test/linalg/bidiag.jl [hunk 1]
```diff
@@ -254,6 +258,12 @@ srand(1)
                     @test Array(op(T, T2)) ≈ op(Tfull, Tfull2)
                 end
             end
+            # test pass-through of A_mul_B! for SymTridiagonal*Bidiagonal
+            TriSym = SymTridiagonal(T.dv, T.ev)
+            @test Array(TriSym*T) ≈ Array(TriSym)*Array(T)
+            # test pass-through of A_mul_B! for AbstractTridiagonal*Bidiagonal
```
Comment (tkelman) [inline test/linalg/bidiag.jl]: should the comment say AbstractTriangular?

Comment (kshyatt) [inline test/linalg/bidiag.jl]: derp, yes

**Review by Sacha0 — APPROVED** [review]
lgtm modulo `@tkelman`'s catch! :)

**Comment by kshyatt** [comment]
`make check-whitespace` passed locally.

**Pull request merged**